CODERABBIT-ISSUES-TO-FIX.md

```
@@ -0,0 +1,186 @@
+# CodeRabbit Issues to Fix - Validated List
+
+**Generated**: 2025-01-27
+**PR**: https://github.com/Beaulewis1977/gemini-cli-tools/pull/1
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)